provider/aws/aws.go

```diff
@@ -219,12 +219,14 @@ type Route53Change struct {
 type Route53Changes []*Route53Change
 
 type profiledZone struct {
-	profile string
-	zone    *route53types.HostedZone
+	profile  string
+	zone     *route53types.HostedZone
+	zoneName string
+	client   Route53API
 }
 
 func (cs Route53Changes) Route53Changes() []route53types.Change {
-	ret := []route53types.Change{}
+	ret := make([]route53types.Change, 0)
```

Comment:
Why? Seems unrelated

Reply:
Yup.. sorry for that its unrelated.. I am not sure if its a better syntax.. I can change it back if you want me to

Reply:
It doesn't matter so just let it like it is.

Reply:
Sure